{
  "version": "4.28.0",
  "time": 232.7,
  "errors": [],
  "tasks": [
    {
      "resource": {
        "name": "ft_compras",
        "title": "Compras",
        "path": "data/ft_compras.csv.gz",
        "schema": {
          "fields": [
            {
              "name": "id_tempo",
              "title": "Tempo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_procedimento",
              "title": "Procedimento de Contratação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_demanda",
              "title": "Órgão Demandante - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_contrato",
              "title": "Órgão do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_proc",
              "title": "Situação do Processo de Compra - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_cont",
              "title": "Situação do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_municipio",
              "title": "Município - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contratado",
              "title": "Contratado - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_tipo_licitacao",
              "title": "Tipo de Licitação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_grupo_matserv",
              "title": "Grupo Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_classe_matserv",
              "title": "Classe Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_material_servico",
              "title": "Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item_matserv",
              "title": "Item Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_processo",
              "title": "Processo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contrato",
              "title": "Número do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_medida",
              "title": "Unidade de Medida - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_linha_fornec",
              "title": "Linha de Fornecimento - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item",
              "title": "Item - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_orc",
              "title": "Unidade Orçamentária - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "unique": false
              }
            },
            {
              "name": "ano_particao",
              "title": "Ano Partição",
              "type": "year",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true
              }
            },
            {
              "name": "dt_item_homologa",
              "title": "Data da Homologação do Item",
              "type": "date",
              "format": "default",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "qt_item_pedido",
              "title": "Item do Pedido - Quantidade",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_referencia",
              "title": "Valor Unitário de Referência",
              "description": "Valor unitário estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_referencia",
              "title": "Valor de Referência",
              "description": "Valor estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_homologado",
              "title": "Valor Unitário Homologado",
              "description": "Valor unitário final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_homologado",
              "title": "Valor Homologado",
              "description": "Valor final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_atualizado",
              "title": "Valor Atualizado",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ".",
              "constraints": {
                "required": true
              }
            }
          ],
          "missingValues": [
            ""
          ],
          "foreignKeys": [
            {
              "fields": [
                "id_processo"
              ],
              "reference": {
                "fields": [
                  "id_processo"
                ],
                "resource": "dm_processo"
              }
            },
            {
              "fields": [
                "id_situacao_cont"
              ],
              "reference": {
                "fields": [
                  "id_situacao_cont"
                ],
                "resource": "dm_situacao_cont"
              }
            },
            {
              "fields": [
                "id_situacao_proc"
              ],
              "reference": {
                "fields": [
                  "id_situacao_proc"
                ],
                "resource": "dm_situacao_proc"
              }
            },
            {
              "fields": [
                "id_linha_fornec"
              ],
              "reference": {
                "fields": [
                  "id_linha_fornec"
                ],
                "resource": "dm_linha_fornec"
              }
            },
            {
              "fields": [
                "id_classe_matserv"
              ],
              "reference": {
                "fields": [
                  "id_classe_matserv"
                ],
                "resource": "dm_classe_matserv"
              }
            },
            {
              "fields": [
                "id_tempo"
              ],
              "reference": {
                "fields": [
                  "id_tempo"
                ],
                "resource": "dm_tempo_diario"
              }
            },
            {
              "fields": [
                "id_municipio"
              ],
              "reference": {
                "fields": [
                  "id_municipio"
                ],
                "resource": "dm_municipio"
              }
            },
            {
              "fields": [
                "id_item_matserv"
              ],
              "reference": {
                "fields": [
                  "id_item_matserv"
                ],
                "resource": "dm_item_matserv"
              }
            },
            {
              "fields": [
                "id_tipo_licitacao"
              ],
              "reference": {
                "fields": [
                  "id_tipo_licitacao"
                ],
                "resource": "dm_tipo_licitacao"
              }
            },
            {
              "fields": [
                "id_item"
              ],
              "reference": {
                "fields": [
                  "id_item"
                ],
                "resource": "dm_item_desp"
              }
            },
            {
              "fields": [
                "id_material_servico"
              ],
              "reference": {
                "fields": [
                  "id_material_servico"
                ],
                "resource": "dm_material_servico"
              }
            },
            {
              "fields": [
                "id_orgao_contrato"
              ],
              "reference": {
                "fields": [
                  "id_orgao_contrato"
                ],
                "resource": "dm_orgao_contrato"
              }
            },
            {
              "fields": [
                "id_contrato"
              ],
              "reference": {
                "fields": [
                  "id_contrato"
                ],
                "resource": "dm_contrato"
              }
            },
            {
              "fields": [
                "id_orgao_demanda"
              ],
              "reference": {
                "fields": [
                  "id_orgao_demanda"
                ],
                "resource": "dm_orgao_demanda"
              }
            },
            {
              "fields": [
                "id_unidade_medida"
              ],
              "reference": {
                "fields": [
                  "id_unidade_medida"
                ],
                "resource": "dm_unidade_medida"
              }
            },
            {
              "fields": [
                "id_grupo_matserv"
              ],
              "reference": {
                "fields": [
                  "id_grupo_matserv"
                ],
                "resource": "dm_grupo_matserv"
              }
            },
            {
              "fields": [
                "id_contratado"
              ],
              "reference": {
                "fields": [
                  "id_contratado"
                ],
                "resource": "dm_contratado"
              }
            },
            {
              "fields": [
                "id_procedimento"
              ],
              "reference": {
                "fields": [
                  "id_procedimento"
                ],
                "resource": "dm_procedimento"
              }
            }
          ]
        },
        "sources": [
          {
            "title": "Tabela origem no schema AGE7 no banco de dados do Portal da Transparência",
            "table": "ft_compras"
          }
        ],
        "format": "csv",
        "mediatype": "text/csv",
        "encoding": "utf-8-sig",
        "dialect": {
          "delimiter": ";"
        },
        "profile": "tabular-data-resource",
        "scheme": "file",
        "hashing": "md5",
        "compression": "gz",
        "stats": {
          "hash": "5f6cceae58b074db2098a70436acb645",
          "bytes": 56252880,
          "fields": 27,
          "rows": 1840639
        }
      },
      "time": 232.7,
      "scope": [
        "hash-count-error",
        "byte-count-error",
        "field-count-error",
        "row-count-error",
        "blank-header",
        "extra-label",
        "missing-label",
        "blank-label",
        "duplicate-label",
        "incorrect-label",
        "blank-row",
        "primary-key-error",
        "foreign-key-error",
        "extra-cell",
        "missing-cell",
        "type-error",
        "constraint-error",
        "unique-error"
      ],
      "partial": false,
      "errors": [
        {
          "cells": [
            "50367",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "563",
            "24316",
            "1",
            "17",
            "379",
            "17564",
            "94354",
            "367262",
            "75173",
            "1968",
            "1831449",
            "2613",
            "2046",
            "2022",
            "2022-04-18",
            "0.0000",
            "250000.0000",
            "250000.00",
            "250000.0000",
            "250000.00",
            "250000.00"
          ],
          "rowNumber": 1798671,
          "rowPosition": 1798672,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75173\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1798672\" violates the foreign key: for \"id_contrato\": values \"75173\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44008",
            "1",
            "1831097",
            "98",
            "1",
            "1",
            "883",
            "12185",
            "1",
            "26",
            "592",
            "21790",
            "181764",
            "369019",
            "75188",
            "1020",
            "1831291",
            "2863",
            "2078",
            "2022",
            "2022-05-13",
            "120.0000",
            "3699.8400",
            "443980.80",
            "3699.8400",
            "443980.80",
            "443980.80"
          ],
          "rowNumber": 1798687,
          "rowPosition": 1798688,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75188\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1798688\" violates the foreign key: for \"id_contrato\": values \"75188\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43956",
            "4",
            "1831017",
            "99",
            "1",
            "1",
            "66",
            "59703",
            "1",
            "58",
            "732",
            "2794",
            "80128",
            "369054",
            "75176",
            "889",
            "1831373",
            "2796",
            "2842",
            "2022",
            "2022-05-13",
            "21.0000",
            "349.7400",
            "7344.54",
            "342.2800",
            "7187.88",
            "7187.88"
          ],
          "rowNumber": 1798986,
          "rowPosition": 1798987,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75176\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1798987\" violates the foreign key: for \"id_contrato\": values \"75176\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43956",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "59703",
            "1",
            "58",
            "732",
            "2794",
            "80128",
            "369037",
            "75176",
            "889",
            "1831373",
            "2796",
            "3331",
            "2022",
            "2022-05-13",
            "21.0000",
            "349.7400",
            "7344.54",
            "342.2800",
            "7187.88",
            "7187.88"
          ],
          "rowNumber": 1799455,
          "rowPosition": 1799456,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75176\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1799456\" violates the foreign key: for \"id_contrato\": values \"75176\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44017",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "21269",
            "1",
            "17",
            "379",
            "9823",
            "94352",
            "369496",
            "75189",
            "889",
            "1831370",
            "2594",
            "1760",
            "2022",
            "2022-05-19",
            "0.0000",
            "848020.8000",
            "848020.80",
            "848020.8000",
            "848020.80",
            "848020.80"
          ],
          "rowNumber": 1799465,
          "rowPosition": 1799466,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75189\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1799466\" violates the foreign key: for \"id_contrato\": values \"75189\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44017",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "21269",
            "1",
            "85",
            "414",
            "7784",
            "158667",
            "369496",
            "75189",
            "889",
            "1831980",
            "3088",
            "1760",
            "2022",
            "2022-05-19",
            "90.0000",
            "277.3600",
            "24962.40",
            "277.3600",
            "24962.40",
            "24962.40"
          ],
          "rowNumber": 1799466,
          "rowPosition": 1799467,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75189\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1799467\" violates the foreign key: for \"id_contrato\": values \"75189\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44018",
            "15",
            "1910409",
            "59",
            "1",
            "1",
            "883",
            "379",
            "1",
            "57",
            "779",
            "2941",
            "5465",
            "369520",
            "75182",
            "190",
            "1831313",
            "2928",
            "2840",
            "2022",
            "2022-05-19",
            "5000.0000",
            "4.8200",
            "24101.50",
            "4.8200",
            "24101.50",
            "24101.50"
          ],
          "rowNumber": 1799469,
          "rowPosition": 1799470,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75182\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1799470\" violates the foreign key: for \"id_contrato\": values \"75182\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44012",
            "15",
            "1831095",
            "96",
            "1",
            "1",
            "844",
            "23493",
            "1",
            "54",
            "502",
            "4032",
            "156228",
            "369127",
            "75178",
            "889",
            "1831296",
            "2944",
            "3293",
            "2022",
            "2022-05-13",
            "17.0000",
            "520.0000",
            "8840.00",
            "520.0000",
            "8840.00",
            "8840.00"
          ],
          "rowNumber": 1800027,
          "rowPosition": 1800028,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75178\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1800028\" violates the foreign key: for \"id_contrato\": values \"75178\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44012",
            "15",
            "1831095",
            "96",
            "1",
            "1",
            "844",
            "23493",
            "1",
            "54",
            "502",
            "14376",
            "155896",
            "369127",
            "75178",
            "889",
            "1831296",
            "2944",
            "3293",
            "2022",
            "2022-05-13",
            "3.0000",
            "760.0000",
            "2280.00",
            "760.0000",
            "2280.00",
            "2280.00"
          ],
          "rowNumber": 1800028,
          "rowPosition": 1800029,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75178\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1800029\" violates the foreign key: for \"id_contrato\": values \"75178\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43956",
            "4",
            "1831071",
            "67",
            "1",
            "1",
            "66",
            "59706",
            "1",
            "77",
            "700",
            "9761",
            "93015",
            "369098",
            "75175",
            "889",
            "1831440",
            "2594",
            "3267",
            "2022",
            "2022-05-13",
            "0.0000",
            "15000.0000",
            "15000.00",
            "10000.0000",
            "10000.00",
            "10000.00"
          ],
          "rowNumber": 1803454,
          "rowPosition": 1803455,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75175\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1803455\" violates the foreign key: for \"id_contrato\": values \"75175\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43956",
            "4",
            "1831071",
            "67",
            "1",
            "1",
            "66",
            "59706",
            "1",
            "77",
            "700",
            "17508",
            "92910",
            "369098",
            "75175",
            "889",
            "1831748",
            "2954",
            "3267",
            "2022",
            "2022-05-13",
            "0.0000",
            "17125.8000",
            "17125.80",
            "16999.9800",
            "16999.98",
            "16999.98"
          ],
          "rowNumber": 1803455,
          "rowPosition": 1803456,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75175\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1803456\" violates the foreign key: for \"id_contrato\": values \"75175\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43968",
            "4",
            "1831105",
            "91",
            "1",
            "1",
            "876",
            "21226",
            "1",
            "13",
            "291",
            "10067",
            "224672",
            "367848",
            "75181",
            "889",
            "1831958",
            "3019",
            "3141",
            "2022",
            "2022-04-27",
            "1.0000",
            "307000.0000",
            "307000.00",
            "280000.0000",
            "280000.00",
            "280000.00"
          ],
          "rowNumber": 1809892,
          "rowPosition": 1809893,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75181\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1809893\" violates the foreign key: for \"id_contrato\": values \"75181\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44008",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "883",
            "33019",
            "1",
            "3",
            "627",
            "3013",
            "226208",
            "368593",
            "75185",
            "889",
            "1831295",
            "2611",
            "3043",
            "2022",
            "2022-05-09",
            "12.0000",
            "4921.1000",
            "59053.20",
            "4921.1000",
            "59053.20",
            "59053.20"
          ],
          "rowNumber": 1811585,
          "rowPosition": 1811586,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75185\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1811586\" violates the foreign key: for \"id_contrato\": values \"75185\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50293",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "386",
            "1",
            "77",
            "73",
            "20205",
            "92481",
            "364603",
            "75187",
            "889",
            "1831761",
            "2872",
            "1760",
            "2021",
            "2022-02-25",
            "0.0000",
            "387942.1700",
            "387942.17",
            "387942.1700",
            "387942.17",
            "387942.17"
          ],
          "rowNumber": 1815073,
          "rowPosition": 1815074,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75187\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1815074\" violates the foreign key: for \"id_contrato\": values \"75187\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44010",
            "1",
            "1831070",
            "66",
            "1",
            "1",
            "66",
            "71130",
            "1",
            "77",
            "533",
            "4338",
            "301708",
            "369566",
            "75171",
            "889",
            "1831751",
            "2807",
            "3083",
            "2022",
            "2022-05-19",
            "0.0000",
            "8500.0000",
            "8500.00",
            "8500.0000",
            "8500.00",
            "8500.00"
          ],
          "rowNumber": 1828312,
          "rowPosition": 1828313,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75171\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1828313\" violates the foreign key: for \"id_contrato\": values \"75171\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44010",
            "1",
            "1831070",
            "66",
            "1",
            "1",
            "66",
            "71130",
            "1",
            "77",
            "533",
            "4338",
            "301708",
            "369566",
            "75171",
            "889",
            "1831751",
            "2807",
            "3083",
            "2022",
            "2022-05-19",
            "0.0000",
            "8500.0000",
            "8500.00",
            "8500.0000",
            "8500.00",
            "8500.00"
          ],
          "rowNumber": 1828312,
          "rowPosition": 1828313,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contratado\": values \"71130\" not found in the lookup table \"dm_contratado\" as \"id_contratado\"",
          "message": "Row at position \"1828313\" violates the foreign key: for \"id_contratado\": values \"71130\" not found in the lookup table \"dm_contratado\" as \"id_contratado\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44002",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "883",
            "29765",
            "1",
            "3",
            "627",
            "19215",
            "275597",
            "368226",
            "75168",
            "889",
            "1831295",
            "2611",
            "3043",
            "2022",
            "2022-05-03",
            "3.0000",
            "4693.4800",
            "14080.44",
            "4693.4800",
            "14080.44",
            "14080.44"
          ],
          "rowNumber": 1829666,
          "rowPosition": 1829667,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75168\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1829667\" violates the foreign key: for \"id_contrato\": values \"75168\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "44015",
            "15",
            "1831095",
            "96",
            "1",
            "1",
            "856",
            "66423",
            "1",
            "9",
            "378",
            "20112",
            "260551",
            "369163",
            "75169",
            "322",
            "1831948",
            "2668",
            "3293",
            "2022",
            "2022-05-16",
            "72.0000",
            "5002.7200",
            "360195.84",
            "5002.7200",
            "360195.84",
            "360195.84"
          ],
          "rowNumber": 1829829,
          "rowPosition": 1829830,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75169\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1829830\" violates the foreign key: for \"id_contrato\": values \"75169\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43975",
            "14",
            "1831009",
            "1",
            "1",
            "1",
            "66",
            "62670",
            "1",
            "51",
            "74",
            "9661",
            "91085",
            "369274",
            "75180",
            "889",
            "1831504",
            "2670",
            "2413",
            "2022",
            "2022-05-17",
            "0.0000",
            "2035.0000",
            "2035.00",
            "2030.0000",
            "2030.00",
            "2030.00"
          ],
          "rowNumber": 1833064,
          "rowPosition": 1833065,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75180\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833065\" violates the foreign key: for \"id_contrato\": values \"75180\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50359",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "66",
            "28327",
            "1",
            "56",
            "527",
            "20558",
            "125204",
            "367613",
            "75170",
            "889",
            "1831940",
            "2814",
            "2046",
            "2022",
            "2022-04-20",
            "3.0000",
            "4390.0000",
            "13170.00",
            "3150.0000",
            "9450.00",
            "9450.00"
          ],
          "rowNumber": 1833536,
          "rowPosition": 1833537,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75170\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833537\" violates the foreign key: for \"id_contrato\": values \"75170\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50359",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "66",
            "68566",
            "1",
            "75",
            "227",
            "5271",
            "132931",
            "367613",
            "75174",
            "889",
            "1831846",
            "3018",
            "2046",
            "2022",
            "2022-04-20",
            "12.0000",
            "2307.1400",
            "27685.68",
            "1875.0000",
            "22500.00",
            "22500.00"
          ],
          "rowNumber": 1833537,
          "rowPosition": 1833538,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75174\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833538\" violates the foreign key: for \"id_contrato\": values \"75174\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50359",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "66",
            "66530",
            "1",
            "60",
            "513",
            "14408",
            "211386",
            "367613",
            "75190",
            "889",
            "1831952",
            "2742",
            "2046",
            "2022",
            "2022-04-20",
            "112.0000",
            "6361.2000",
            "712454.40",
            "6214.0000",
            "695968.00",
            "695968.00"
          ],
          "rowNumber": 1833538,
          "rowPosition": 1833539,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75190\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833539\" violates the foreign key: for \"id_contrato\": values \"75190\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50359",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "66",
            "66530",
            "1",
            "54",
            "424",
            "9312",
            "125444",
            "367613",
            "75190",
            "889",
            "1831296",
            "2944",
            "2046",
            "2022",
            "2022-04-20",
            "3.0000",
            "912.0000",
            "2736.00",
            "912.0000",
            "2736.00",
            "2736.00"
          ],
          "rowNumber": 1833544,
          "rowPosition": 1833545,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75190\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833545\" violates the foreign key: for \"id_contrato\": values \"75190\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50359",
            "4",
            "1831079",
            "76",
            "1",
            "1",
            "66",
            "21480",
            "1",
            "54",
            "427",
            "3990",
            "147012",
            "367613",
            "75179",
            "889",
            "1831296",
            "2944",
            "2046",
            "2022",
            "2022-04-20",
            "15.0000",
            "1380.0000",
            "20700.00",
            "1380.0000",
            "20700.00",
            "20700.00"
          ],
          "rowNumber": 1833545,
          "rowPosition": 1833546,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75179\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1833546\" violates the foreign key: for \"id_contrato\": values \"75179\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43969",
            "15",
            "1831008",
            "5",
            "1",
            "1",
            "856",
            "66423",
            "1",
            "9",
            "378",
            "20112",
            "260549",
            "366291",
            "75186",
            "322",
            "1831948",
            "2668",
            "2925",
            "2022",
            "2022-03-31",
            "288.0000",
            "2460.9200",
            "708744.96",
            "2460.9200",
            "708744.96",
            "708744.96"
          ],
          "rowNumber": 1834355,
          "rowPosition": 1834356,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75186\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1834356\" violates the foreign key: for \"id_contrato\": values \"75186\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "55950",
            "1",
            "77",
            "273",
            "14784",
            "90078",
            "368718",
            "75191",
            "889",
            "1831537",
            "2954",
            "1760",
            "2022",
            "2022-05-10",
            "0.0000",
            "163625.8900",
            "163625.89",
            "41955.3600",
            "41955.36",
            "41955.36"
          ],
          "rowNumber": 1837554,
          "rowPosition": 1837555,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75191\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1837555\" violates the foreign key: for \"id_contrato\": values \"75191\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50345",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "65933",
            "1",
            "54",
            "502",
            "6435",
            "238653",
            "368280",
            "75177",
            "889",
            "1831296",
            "2944",
            "1760",
            "2022",
            "2022-05-10",
            "1125.0000",
            "359.7200",
            "404685.00",
            "286.0000",
            "321750.00",
            "321750.00"
          ],
          "rowNumber": 1837556,
          "rowPosition": 1837557,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75177\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1837557\" violates the foreign key: for \"id_contrato\": values \"75177\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43989",
            "4",
            "1831094",
            "95",
            "1",
            "1",
            "637",
            "69374",
            "1",
            "77",
            "808",
            "4383",
            "91625",
            "368937",
            "75183",
            "889",
            "1899033",
            "2869",
            "3043",
            "2022",
            "2022-05-12",
            "0.0000",
            "114220.0000",
            "114220.00",
            "48475.0000",
            "48475.00",
            "48475.00"
          ],
          "rowNumber": 1837636,
          "rowPosition": 1837637,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75183\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1837637\" violates the foreign key: for \"id_contrato\": values \"75183\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43989",
            "4",
            "1831094",
            "95",
            "1",
            "1",
            "637",
            "69374",
            "1",
            "77",
            "808",
            "4383",
            "91625",
            "368937",
            "75183",
            "889",
            "1899033",
            "2869",
            "3043",
            "2022",
            "2022-05-12",
            "0.0000",
            "36600.0000",
            "36600.00",
            "53461.0000",
            "53461.00",
            "53461.00"
          ],
          "rowNumber": 1837638,
          "rowPosition": 1837639,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75183\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1837639\" violates the foreign key: for \"id_contrato\": values \"75183\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43995",
            "14",
            "1831041",
            "41",
            "1",
            "1",
            "883",
            "68269",
            "1",
            "69",
            "446",
            "21581",
            "170046",
            "368985",
            "75172",
            "1544",
            "1831312",
            "2538",
            "2689",
            "2022",
            "2022-05-12",
            "100.0000",
            "20.9900",
            "2099.00",
            "18.5000",
            "1850.00",
            "1850.00"
          ],
          "rowNumber": 1839904,
          "rowPosition": 1839905,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75172\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1839905\" violates the foreign key: for \"id_contrato\": values \"75172\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50268",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "5470",
            "1",
            "51",
            "383",
            "17638",
            "94053",
            "368991",
            "75184",
            "981",
            "1831360",
            "2670",
            "2014",
            "2021",
            "2022-05-12",
            "0.0000",
            "71800.0000",
            "71800.00",
            "39299.7600",
            "39299.76",
            "39299.76"
          ],
          "rowNumber": 1840330,
          "rowPosition": 1840331,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "for \"id_contrato\": values \"75184\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "message": "Row at position \"1840331\" violates the foreign key: for \"id_contrato\": values \"75184\" not found in the lookup table \"dm_contrato\" as \"id_contrato\"",
          "description": "Values in the foreign key fields should exist in the reference table"
        }
      ],
      "stats": {
        "errors": 31
      },
      "valid": false
    }
  ],
  "stats": {
    "errors": 31,
    "tasks": 1
  },
  "valid": false
}